torch_ipex/csrc/cpu/dbl/Common.cpp

```diff
@@ -17,9 +17,13 @@ namespace dbl {
 namespace comm {
 
 dil::tensor dil_tensor_from_dense(const at::Tensor& tensor) {
-  AT_ASSERTM(
+  TORCH_CHECK(
     tensor.layout() == at::Layout::Strided,
-    "dil_tensor_view_from_dense expects dense tensor input");
+    "dil_tensor_from_dense expects dense tensor input");
+  TORCH_CHECK(tensor.scalar_type() == at::ScalarType::Float
+             "dil_tensor_from_dense expects  tensor input");
```

Comment:
`dil_tensor_from_dense expects tensor of at::ScalarType::Float`

Reply:
Done